src/coreclr/vm/object.h

```diff
@@ -493,7 +493,7 @@ inline void ClearObjectReference(OBJECTREF* dst)
 
 // CopyValueClass sets a value class field
 
-void STDCALL CopyValueClassUnchecked(void* dest, void* src, MethodTable *pMT);
+void CopyValueClassUnchecked(void* dest, void* src, MethodTable *pMT);
 void STDCALL CopyValueClassArgUnchecked(ArgDestination *argDest, void* src, MethodTable *pMT, int destOffset);
```

Comment:
```suggestion
void CopyValueClassArgUnchecked(ArgDestination *argDest, void* src, MethodTable *pMT, int destOffset);
```
Looks like STDCALL was copy&pasted unnecessarily